doc_user/UserManual.adoc

```diff
@@ -801,6 +801,288 @@ image:image251.png[success]
+
+The *Inventory table* in the center identifies the inventory by:
+
+* *Reference*: indicates the reference of the ward inventory.
+* *Ward*: indicates the ward related to the inventory.
+* *Date*: indicates the date with which the ward inventory was entered into the system.
+* *Status*: indicates the status of the ward inventory.
+* *User*: indicates the user who entered the ward inventory.
+
+The *Buttons panel* has all the functions of the ward inventory, the following choices are available:
+
+* *[.underline]##N##ew*: this button is used to create a new ward inventory.
+* *[.underline]##E##dit*: this button is used to modify an existing ward inventory.
+* *[.underline]##V##iew*: this button is used to show an existing ward inventory.
+* *[.underline]##D##elete*: this button is used to cancel an existing ward inventory.
+* *[.underline]##C##lose*: this button is used to close the window and return to the main menu.
+
+image:image262.png[Ward Inventory Browser]
+
+[#create-a-new-ward-inventory]
+===== 4.5.2.1 Create a New ward Inventory ([.underline]##N##ew)
+
+The *New Inventory* function is used to create a new inventory on the main store. Click on *[.underline]##N##ew* button in the *_Manage Ward Inventory_* window to access the *_New Inventory_* window.
+
+image:image263.png[New ward Inventory]
+
+After the selection of the ward (FEMALE WARD in this example), the *_New Inventory_* window is shown, allowing for the management of the ward inventory of the FEMALE WARD:
+
+image:image264.png[New ward Inventory]
+
+The *_New Inventory_* window is composed of three areas: the *Inventory panel* at the top, the *Inventory row table* in the center and the *Buttons panel* on the bottom.
+
+The data shown in the *Upper panel* area are:
+
+* *Ward*: the ward that we are making the inventory.
+* *Date*: the date of the inventory.
+* *Reference*: the reference associated to the inventory.
+
+The data shown in the *Center panel* area represente inventory row. they are:
+
+* *Code*: the code of the phamaceutical.
+* *Product*: the description of the phamaceutical.
+* *New Lot*: indicates if the inventory row is for a new lot or not.
+* *Lot's Code.*: the lot code of the pharmaceutical.
+* *Due Date*: the due date of the lot.
+* *Theoretic Qty*: the theoretic quantity of the lot of the pharmaceutical.
+* *Real Qty*: the real(physical) quantity of the lot of the pharmaceutical.
+* *Total Cost*: the total cost that represents the real value of the lot.
+
+The *Buttons panel* is used to perform some action:
+
+* *[.underline]##S##ave*: this button is used to save the inventory.
+* *[.underline]##L##ot*: this button is used to link new lot to a new specific product (the inventory row that column *New Lot* as value *N*). to perform that, first highlighted the product and click on this button.
+
+NOTE: This button is enabled just for the inventory row that column *New Lot* has value *N* else the button is disable.
```

Comment:
```suggestion
NOTE: This button is enabled only for the inventory row where the column *New Lot* has value the value *N*, otherwise the button is disabled.
```